src/python/kiss_icp/tools/cmd.py

```diff
@@ -150,6 +153,13 @@ def kiss_icp_pipeline(
         help="[Optional] Specify if you want to start to process scans from a given starting point",
         rich_help_panel="Additional Options",
     ),
+    meta: Optional[Path] = typer.Option(
```

Comment:
If you still think it is necessary to specify the metadata file from the CLI then I'd add this to be consistent (no need to do it yourself ;)) :

```diff
diff --git a/src/python/kiss_icp/tools/cmd.py b/src/python/kiss_icp/tools/cmd.py
index 04d7b7b..80eba07 100644
--- a/src/python/kiss_icp/tools/cmd.py
+++ b/src/python/kiss_icp/tools/cmd.py
@@ -156,7 +156,9 @@ def kiss_icp_pipeline(
     meta: Optional[Path] = typer.Option(
         None,
         "--meta",
+        "-m",
         exists=True,
+        show_default=False,
         help="[Optional] For Ouster pcap dataloader, you need to specify a metadata json file",
         rich_help_panel="Additional Options",
     ),
```

Reply:
Yep, good point that it's better to not ask for explicit `--meta` param if we can resolve the file, so I've added the `find_metadata_json()` that attempts to do this and set `meta` param as optional in CLI.

However I still left it as an optional param, because sometime files can be located separately and there is no standard tools that creates such Ouster pcap data files + metadata.json together, and it means users may have really different setups.

But for majority of cases just calling `kiss_icp_pipeline --visualize <path-to-ouster.pcap>` should do the job.